src/libs/Navigation/AppNavigator/Navigators/RightModalNavigator.js

```diff
@@ -20,10 +20,18 @@ function RigthModalNavigator() {
             <Stack.Screen
                 name="NewGroup"
                 component={ModalStackNavigators.NewGroupModalStackNavigator}
+                options={{
+                    // Disable animation for this screen because it causes an animation glitch when using shortcuts
```

Comment:
Do we have a link to a video with the glitch? I think it'd be nice to add it here so we have a reference in case we wanna fix it later on.

Reply:
Adding video here: 
https://github.com/Expensify/App/assets/36083550/62c85ae9-72a7-4819-a1b6-278e2a397dd7